<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KOnlineJobOutboxView</class>
 <widget class="QWidget" name="KOnlineJobOutboxView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>374</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="2,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="5">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="KMMEmptyTreeView" name="m_onlineJobView">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KMMSearchWidget" name="m_searchWidget" native="true"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="m_buttonSend">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="m_buttonNewCreditTransfer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>New credit transfer</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="m_buttonEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="m_buttonRemove">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>208</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMMSearchWidget</class>
   <extends>QWidget</extends>
   <header>kmmsearchwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KMMEmptyTreeView</class>
   <extends>QTreeView</extends>
   <header>kmmemptyview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_onlineJobView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <signal>sendQueue()</signal>
 </slots>
</ui>
